subsys/testsuite/ztest/Kconfig

```diff
@@ -33,6 +33,7 @@ config ZTEST_ASSERT_VERBOSE
 
 config ZTEST_THREAD_PRIORITY
 	int "Testing thread priority"
+	default -2 if !PREEMPT_ENABLED
 	default -1
 	help
 	  Set priority of the testing thread. Default is -1 (cooperative).
```

Comment:
I didn't update the help text. If I should please provide a suggestion.

Reply:
thinking about it a bit more...  if ztest needs cooperative threading we probably need to depend on it, then we can drop this config completely and just use K_LOWEST_APPLICATION_THREAD_PRIO when creating the thread

Reply:
I never had a need to use ztest thread priority other than the default. Also looking at the tests folder, there are no test cases that would redefine the default value. This may be well one of those Kconfig options that is not used and not needed. However, since we would remove functionality which was there before it should probably be discussed in a larger group. Maybe dev-review meeting?